### PR TITLE
[binding]: add decode opts

### PR DESCRIPTION
**Pull request by Mddct**

### CODE REVIEW

File: runtime/core/api/wenet_api.cc [hunk 1]
```diff
@@ -92,6 +92,14 @@ class Recognizer {
     }
     resource_->post_processor =
         std::make_shared<wenet::PostProcessor>(*post_process_opts_);
+
+    // Set decode opts
+    decode_options_->chunk_size = chunk_size_;
+    decode_options->num_left_chunks = num_left_chunks_;
```
Comment (pengzhendong) [inline runtime/core/api/wenet_api.cc:98]: decode_options_

**Review by pengzhendong — CHANGES_REQUESTED** [review]

File: runtime/core/api/wenet_api.cc (hunk 1 above)
Comment (Mddct) [reply runtime/core/api/wenet_api.cc]: done

File: runtime/binding/python/cpp/binding.cc [hunk 2]
```diff
@@ -35,4 +35,6 @@ PYBIND11_MODULE(_wenet, m) {
   m.def("wenet_set_context_score", &wenet_set_context_score,
         "set context bonus score");
   m.def("wenet_set_language", &wenet_set_language, "set language");
+  m.def("wenet_set_chunk_size", &wenet_set_chunk_size, "set chunk size");
+  m.def("wenet_set_weights, &wenet_set_weights, "set decode weights");
```
Comment (pengzhendong) [inline runtime/binding/python/cpp/binding.cc:39]: 漏了个引号

**Review by pengzhendong — CHANGES_REQUESTED** [review]

File: runtime/binding/python/cpp/binding.cc (hunk 2 above)
Comment (Mddct) [reply runtime/binding/python/cpp/binding.cc]: done

**Review by pengzhendong — APPROVED** [review]

**Pull request merged**